packages/amplication-client/src/Assistant/Assistant.tsx

```diff
@@ -53,6 +63,16 @@ const WIDTH_STATE_SETTINGS: Record<
 const Assistant = () => {
   const { currentWorkspace } = useAppContext();
 
+  const { stigg } = useStiggContext();
+
+  const { hasAccess } = stigg.getMeteredEntitlement({
+    featureId: BillingFeature.JovuRequests,
+  });
+
+  const { data } = useQuery(GET_CONTACT_US_LINK, {
```

Comment:
With all the recent changes in meeting URLs (yours, mine, Paz, Oren) in the product, this specifically will get you the meeting URL that is defined in the server's .env which is currently Oren's meeting link, but Paz asked me to change it to his...
So - which meeting URL do you want/expect here?

Reply:
let's merge this PR and we can align again on next one

Reply:
Okay, as long as you don't mind which exact meeting link will be here (Paz/Oren)